src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
```

Comment:
**question:** what's your reasoning behind using this pre-v5 component instead of just plain CSS?

Reply:
sorry, I thought that `Text2Xs, TextXs` are the base style which different with `SubTitle, Tittle, .etc`